storage/reader.go

```diff
@@ -149,7 +149,13 @@ func (o *ObjectHandle) NewRangeReader(ctx context.Context, offset, length int64)
 			req.Header.Set("Range", fmt.Sprintf("bytes=%d-%d", start, offset+length-1))
 		}
 		// We wait to assign conditions here because the generation number can change in between reopen() runs.
-		req.URL.RawQuery = conditionsQuery(gen, o.conds)
```

Comment:
I think conditionsQuery is no longer used anywhere in the library? If so we can remove it since it's not exported.

Reply:
Thanks for catching that. Removed.